pkg/kyma/cmd/test/common.go

```diff
@@ -0,0 +1,93 @@
+package test
+
+import (
+	"fmt"
+	"io"
+
+	oct "github.com/kyma-incubator/octopus/pkg/apis/testing/v1alpha1"
+	"github.com/kyma-project/cli/internal/kube"
+	"github.com/kyma-project/cli/pkg/kyma/cmd/test/client"
+	"github.com/olekukonko/tablewriter"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const NamespaceForTests = "kyma-system"
+
+func ListTestDefinitionNames(cli client.TestRESTClient) ([]string, error) {
+	defs, err := cli.ListTestDefinitions()
+	if err != nil {
+		return nil, fmt.Errorf("unable to list test definitions. E: %s", err.Error())
+	}
+
+	var result = make([]string, len(defs.Items))
+	for i := 0; i < len(defs.Items); i++ {
+		result[i] = defs.Items[i].GetName()
+	}
+	return result, nil
+}
+
+func ListTestSuiteNames(cli client.TestRESTClient) ([]string, error) {
+	suites, err := cli.ListTestSuites()
+	if err != nil {
+		return nil, fmt.Errorf("unable to list test suites. E: %s", err.Error())
+	}
+
+	var result = make([]string, len(suites.Items))
+	for i := 0; i < len(suites.Items); i++ {
+		result[i] = suites.Items[i].GetName()
+	}
+	return result, nil
+}
+
+func ListTestSuitesByName(cli client.TestRESTClient, names []string) ([]oct.ClusterTestSuite, error) {
+	suites, err := cli.ListTestSuites()
+	if err != nil {
+		return nil, fmt.Errorf("unable to list test suites. E: %s", err.Error())
+	}
+
+	result := []oct.ClusterTestSuite{}
+	for _, suite := range suites.Items {
+		for _, tName := range names {
+			if suite.ObjectMeta.Name == tName {
+				result = append(result, suite)
+			}
+		}
+	}
+	return result, nil
+}
+
+func NewTestSuite(name string) *oct.ClusterTestSuite {
+	return &oct.ClusterTestSuite{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "testing.kyma-project.io/v1alpha1",
+			Kind:       "ClusterTestSuite",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: NamespaceForTests,
+			Labels: map[string]string{
+				"requires-testing-bundle": "true",
```

Comment:
Why that labels are needed by default? If a test definition does not require the testing bundle it should not get executed?

Reply:
Fixed